pages/common/stat.md

```diff
@@ -5,24 +5,24 @@
 
 - Show file properties such as size, permissions, creation and access dates among others:
```

Comment:
The following edit can be applied later in another PR:

```suggestion
- Show a specific file properties such as size, permissions, creation and access dates among others:
```

All descriptions in all pages should be standardized.

Reply:
This and three suggestions below are implemented  in #9438